src/editor-sidebar/metadata-editor-modal.js

```diff
@@ -230,6 +239,45 @@ Plugin Description`,
 						setTheme( { ...theme, recommended_plugins: value } )
 					}
 				/>
+
+				<TextareaControl
+					label={ __( 'Font credits', 'create-block-theme' ) }
+					help={
+						<>
+							<Button
+								variant="secondary"
+								onClick={ updateFontCredits }
+							>
+								{ __(
+									'Get updated font credits',
+									'create-block-theme'
+								) }
+							</Button>
+							<br />
+							{ __(
+								'Credits and licensing information for fonts used in the theme.',
+								'create-block-theme'
+							) }
+							<br />
+							<ExternalLink href="https://make.wordpress.org/themes/handbook/review/required/#1-licensing-copyright">
+								{ __( 'Read more.', 'create-block-theme' ) }
+							</ExternalLink>
+						</>
+					}
+					// eslint-disable-next-line @wordpress/i18n-no-collapsible-whitespace
+					placeholder={ __(
+						`Font Name
+Copyright
+License
+Source`,
+						'create-block-theme'
+					) }
```

Comment:
Translate each part individually instead.

```suggestion
					placeholder={ `${ __( 'Font Name', 'create-block-theme' ) }
${ __( 'Copyright', 'create-block-theme' ) }
${ __( 'License', 'create-block-theme' ) }
${ __( 'Source', 'create-block-theme' ) }` }
```

Also, perhaps modify to clarify how each part of the license should look like.